Code/Tools/FBuild/FBuildCore/Protocol/Client.cpp

```diff
@@ -451,6 +805,7 @@ void Client::Process( const ConnectionInfo * connection, const Protocol::MsgJobR
     ServerState * ss = (ServerState *)connection->GetUserData();
     ASSERT( ss );
 
+    AStackString<> workerName( ss->m_RemoteName );
```

Comment:
We're now making a copy of this name? Is that needed? (I don't see anything in the code that would indicate that, so I feel I may be missing something)